cyclops-ctrl/internal/models/dto/k8s.go

```diff
@@ -291,6 +291,57 @@ func (p *PersistentVolumeClaim) SetDeleted(deleted bool) {
 	p.Deleted = deleted
 }
 
+type Secret struct {
+	Group     string            `json:"group"`
+	Version   string            `json:"version"`
+	Kind      string            `json:"kind"`
+	Name      string            `json:"name"`
+	Namespace string            `json:"namespace"`
+	Type      string            `json:"type"`
+	Data      map[string][]byte `json:"data"`
```

Comment:
Can you send only the keys from the secret without values? I would like to allow users to check which secret keys can they use, but without exposing the secret value

Reply:
Cool, is this the response you are expecting?
```json
{
    "group": "",
    "version": "v1",
    "kind": "Secret",
    "name": "justsecret-2",
    "namespace": "cyclops",
    "type": "Opaque",
    "dataKeys": [
        "KEY"
    ],
    "deleted": false
}
```

Reply:
Yeah, looks good